build/plugins/inline-fs.js

```diff
@@ -64,6 +64,22 @@ async function inlineFs(code, filepath) {
   for (const foundIndex of foundIndices) {
     if (foundIndex === undefined) continue; // https://github.com/microsoft/TypeScript/issues/36788
 
+    // First iterate backwards until a newline to determine if we are in a block comment.
+    let maybeInBlockComment = false;
+    for (let i = foundIndex; i > 0; i--) {
+      if (code[i] === '*') {
```

Comment:
I'm just realizing this is horribly tailored to the one case I found. Doesn't catch all block comments (don't need an asterisk on every line...), and trips up on multiplication. But how can we do something similar w/o parsing the AST?

Reply:
We could drop trying to detect if inside a comment, go back to this being a warning, and iterate all warnings and throw on any `ENOENT` that isn't one we expect.

Reply:
This entire file seems tailored to our use cases. If we aren't handling one of these cases properly, will this plugin fail loudly?

Reply:
Yes, it'd be throwing an error any failing the build step.